src/control/cmd/daos/acl.go

```diff
@@ -405,6 +405,10 @@ func (cmd *containerSetOwnerCmd) Execute(args []string) error {
 			cmd.ContainerID())
 	}
 
+	if err := dfsError(C.dfs_root_chown(ap.cont, user, group)); err != nil {
+		return errors.Wrapf(err, "%s failed", fsOpString((ap.fs_op)))
+	}
```

Comment:
Should this be part of daos_cont_set_owner, so it is always called when the owner is changed, not just from the daos tool? Might also make the #include issue easier to resolve.

Reply:
i thought about this, but im not seeing a way we can call DFS from a container module.
DFS relies on container module but not the other way around.

Reply:
what we can add is a new function in dfs that calls daos_cont_set_owner and we call it dfs_cont_set_owner. let me explore that

Reply:
I like that solution. Looks good!